pyanaconda/packaging/rpmostreepayload.py

```diff
@@ -292,10 +292,11 @@ def postInstall(self):
         # However, we ignore the case where the remote already exists,
         # which occurs when the content itself provides the remote
         # config file.
-        sysroot = OSTree.Sysroot.new(self._sysroot_path)
+        sysroot_path = Gio.File.new_for_path(iutil.getSysroot())
+        sysroot = OSTree.Sysroot.new(sysroot_path)
```

Comment:
I have no idea what's going on in this patch, but maybe ``sysroot_file`` would be a better name for the variable?